01_fib/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+
+	n1 := 0
+	n2 := 1
+
+	fibSeries := ""
+
+	for i := 0; i < n; i++ {
+		if n1 == 0 {
+			fibSeries = fmt.Sprintf("%s%d", fibSeries, n1)
```

Comment:
fibSeries is guaranteed to be `""` at this point. No need to include in the `fmt.Sprintf()` call.

Reply:
Good point and as you suggested in another comment, initialised the `fibSeries` to `"0"` and the loop now starts from `1` so that we don't need to handle the edge case in the loop.